fdrs_functions.h

```diff
@@ -494,12 +553,12 @@ void releaseLoRa(uint8_t interface) {
         for (int i = 0; i < lenLORAG; i++) {
           if ( j > lora_size) {
             j = 0;
-            transmitLoRa(broadcast_mac, thePacket, j);
+            transmitLoRa(&loraBroadcast, thePacket, j);
           }
           thePacket[j] = LORAGbuffer[i];
           j++;
         }
-        transmitLoRa(broadcast_mac, thePacket, j);
+        transmitLoRa(&loraBroadcast, thePacket, j);
```

Comment:
This is a quick shot in the dark, but here you changed it to pass an address instead of the variable. That's not your issue is it? I'm still shaky with *s and &s to be honest.

Reply:
When I changed the MAC addresses from 3 bytes to 2 bytes I changed some of the types from an array of uint8_t to a single uint16_t.   Because of that change I had to change the way the broadcast address is passed into that transmitLoRa function.  Since the address is no longer an array I had to add the Address of (&) in front of the variable in order to pass in the pointer.  I don't think it has any effect on why the data isn't matching.   I'd have to spend more time on it but there is something going on as far as the byte order - big endian vs little endian.  Looking at the output in the other thread the byte order is reversed.   But there is also something else going on.  Not sure but I'll have to look at it some other time.

Reply:
I figured there was a good reason, I just wanted to check. Hey have a nice vacation!